src/components/TimeEstimator/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Time estimator.
```

Comment:
`TimeEstimator` is missing an entry point here in the index, and then the component defined in `.../components/TimeEstimator/TimeEstimator.js`

Reply:
This is another thing I've been thinking about and questioning, and I'd like to hear your thoughts - if we only ever intend to export a simple component, do you think an entry point is necessary?

Reply:
Yeah it does seem like overkill in some circumstances, like this one 😅 
As far as I know, there aren't any tests associated with this structure? That would be my only hesitation.

But honestly I don't feel strongly either way! So I'm cool with this approach. 👍